modules/AlphaWalletFoundation/AlphaWalletFoundation/Extensions/Data.swift

```diff
@@ -22,18 +22,6 @@ extension Data {
         return "0x" + self.hex()
     }
 
-    public init(_hex value: String, chunkSize: Int) {
```

Comment:
But didn't we add this to avoid a crash with long `value`? How about keeping it but moving `chuckSize` as a local? We'll need to adjust the `init()` name since there will be 2 `init(_hex:)`. Maybe make this as `init(_hex:)` with a local `let chunkSize = 100` and make the current `init(_hex:)` private and rename it slightly?

Reply:
not quite sure if it wil work as expected,  tried with small chunk size value and it returns not same result as without chunking. Maybe its because it was'nt an odd number, not sure.

Reply:
Oh right, if it's odd number `.init(_hex:)` just drops it (which is wrong), but we can fix it separately.